src/script_opt/Inline.cc

```diff
@@ -139,8 +140,15 @@ void Inliner::InlineFunction(FuncInfo* f)
 	// particular body.
 	curr_frame_size = f->Scope()->Length();
 
+	auto oi = f->Body()->GetOptInfo();
```

Comment:
Out of curiosity to help me understand the changes, why does this save off the values before calling `Inline()`? Is it because they might change further inside the call to inline that statement, and we want to reset them back to what they were before we continue with later statements?

Reply:
Perfect, that was the explanation I was looking for 👍